tests/test_joint_logprob.py

```diff
@@ -229,6 +230,7 @@ def test_persist_inputs():
     assert y_vv in ancestors([logp_2])
 
 
+@pytest.mark.skip(reason="Disabled until we decide how we want this to work.")
```

Comment:
Wouldn't we want the logdensity to be bound to `x_rv`, i.e. it will use a different realization of `x` at each call?

Reply:
Yeah, this is an elective warning, because we definitely need to support the derivation of log-densities with un-bound/valued `RandomVariable` terms.

Reply:
If this is something we want to support we should remove this warning, and instead document the behavior in the docstring (it's already the case).